app/frontend/src/features/search/SearchPage.tsx

```diff
@@ -62,12 +55,18 @@ export default function SearchPage() {
   );
 
   const { query } = useParams<{ query?: string }>();
+
+  const showResults = useRef(false);
   useEffect(() => {
-    setTimeout(
-      () => map.current?.resize(),
-      theme.transitions.duration.standard
-    );
-  }, [query, theme.transitions.duration.standard]);
+    const shouldShowResults = !!query || !!selectedResult;
+    if (showResults.current !== shouldShowResults) {
+      showResults.current = shouldShowResults;
+      setTimeout(
+        () => map.current?.resize(),
```

Comment:
Is this Mapbox magic that works out the right zoom level when the number of users available on the map changes?

Reply:
No - it's just necessary to call when the size of the canvas changes, otherwise you have tiles which aren't loaded

Reply:
And the timeout is because of the collapse animation, to make sure happens at the end